social_core/backends/google_openidconnect.py

```diff
@@ -12,10 +12,3 @@ class GoogleOpenIdConnect(GoogleOAuth2, OpenIdConnectAuth):
     # differs from value in discovery document
     # http://openid.net/specs/openid-connect-core-1_0.html#rfc.section.15.6.2
     ID_TOKEN_ISSUER = 'accounts.google.com'
-
-    def user_data(self, access_token, *args, **kwargs):
```

Comment:
@aleksanb I am curious to know how to obtain userdata if this code is removed? Someone just made another PR to use new APIs to get userdata. [PR Link](https://github.com/python-social-auth/social-core/pull/330)